Modules/overlapped.c

```diff
@@ -1806,7 +1806,7 @@ _overlapped_Overlapped_WSASendTo_impl(OverlappedObject *self, HANDLE handle,
         case ERROR_IO_PENDING:
             Py_RETURN_NONE;
         default:
-            self->type = TYPE_NOT_STARTED;
+            Overlapped_clear(self);
```

Comment:
I was very confused with this, I thought it was `tp_clear` but it's explicitly not `tp_clear` so this looks like the correct fix.